darwin/dataset/utils.py

```diff
@@ -158,41 +164,55 @@ def _stratify_samples(idx_to_classes, split_seed, test_percentage, val_percentag
     X_train, X_val, X_test : list
         List of indices of the images for each split
     """
+    # Import locally sklearn as to prevent making it a requirement globally for darwin-py
+    from sklearn.model_selection import train_test_split
+
     # Expand the list of files with all the classes
     expanded_list = [(k, c) for k, v in idx_to_classes.items() for c in v]
     # Stratify
     file_indices, labels = zip(*expanded_list)
     X_train, X_tmp, y_train, y_tmp = remove_cross_contamination(
-        *train_test_split(np.array(file_indices), np.array(labels),
-                          test_size=val_percentage + test_percentage,
-                          random_state=split_seed,
-                          stratify=labels)
+        *train_test_split(
+            np.array(file_indices),
+            np.array(labels),
+            test_size=int((val_percentage + test_percentage) * 100) / 100,
```

Comment:
why do we need to truncate the decimals for test_size?

Reply:
because of funny handling of floating points numbers (happens on Nest for example, with 30 images class balanced)